src/main/common/router/routablePath.ts

```diff
@@ -33,6 +35,10 @@ export class RoutablePath {
     return `${featureName}/views/${viewPath}`
   }
 
+  static isValidParameterValue (parameterValue: string): boolean {
```

Comment:
That makes this method public. Please move it outside class.

Reply:
you can also make it `private`

Reply:
done